### PR TITLE
Feature/wdp190703 5/add class active on buttons

### PR DESCRIPTION
**Pull request by DzesikaTkaczyk**

### CODE REVIEW

File: src/sass/components/_button.scss [hunk 1]
```diff
@@ -33,4 +33,21 @@
...
+
+.btn-active {
+  display: inline-block;
+  padding: 5px 10px;
+  border: 1px solid #d58e32;
+  color: #d58e32;
+  font-size: 13px;
+  line-height: 22px;
+  &:hover {
+    background-color: #2a2a2a;
```
Comment (matwjablonski) [inline src/sass/components/_button.scss]: move colors to variables

File: src/sass/components/_button.scss [hunk 2]
```diff
@@ -33,4 +33,21 @@
...
+}
+
+.btn-active {
+  display: inline-block;
+  padding: 5px 10px;
+  border: 1px solid #d58e32;
+  color: #d58e32;
+  font-size: 13px;
+  line-height: 22px;
+  &:hover {
```
Comment (matwjablonski) [inline src/sass/components/_button.scss]: use @media to add hover only when device supports hover

File: src/sass/components/_button.scss [hunk 3]
```diff
@@ -33,4 +33,21 @@
...
+
+  &:active {
+    color: #d58e32;
+  }
+}
+
+.btn-active {
+  display: inline-block;
+  padding: 5px 10px;
+  border: 1px solid #d58e32;
```
Comment (matwjablonski) [inline src/sass/components/_button.scss]: color to variables

File: src/sass/components/_button.scss (hunk 1 above)
Comment (DzesikaTkaczyk) [reply src/sass/components/_button.scss]: zrobiłam, nie wiem czy hovery są do końca cacy, wydaje mi się ze są ale...nom

File: src/sass/_my_mixins.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@mixin hover() {
+  @media not all and (hover: none) {
```
Comment (matwjablonski) [inline src/sass/_my_mixins.scss]: don't use your own files, as `my_something` 

Daniel in prev PR `https://github.com/matwjablonski/kodilla-wdp-1907-03/pull/2` - you can use his solution :)

File: src/sass/components/_button.scss [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
   font-weight: 600;
   padding: 10px 30px;
 
-  &:not(.no-hover):hover {
-    color: #ffffff !important;
-    background-color: $primary;
-    text-decoration: none;
+  @include hover {
+    &:not(.no-hover):hover {
+      color: $color-white !important;
```
Comment (matwjablonski) [inline src/sass/components/_button.scss]: don't use important

File: src/sass/components/_button.scss [hunk 6]
```diff
@@ -24,30 +26,30 @@
...
-    background-color: #2a2a2a;
-    color: #ffffff;
+  @include hover {
+    background-color: $color-black;
+    color: $color-white;
   }
 
   &:active {
-    color: #d58e32;
+    color: $color-active;
```
Comment (matwjablonski) [inline src/sass/components/_button.scss]: please use same variables as your team mates ;)

File: src/sass/_my_mixins.scss (hunk 4 above)
Comment (DzesikaTkaczyk) [reply src/sass/_my_mixins.scss]: zrobione

**Comment by matwjablonski** [comment]
check your builds and fix problems ;)

**Comment by matwjablonski** [comment]
build doesn't pass

please resolve conflicts

**Comment by matwjablonski** [comment]
@DzesikaTkaczyk please fix builds. More info you will find when you click "Details", and then "Logs"

**Pull request merged**